app/retail/emails.py

```diff
@@ -972,45 +972,45 @@ def render_new_bounty_roundup(to_email):
 </p>
 '''
     highlights = [{
-        'who': 'dcd018',
+        'who': 'KapsonLabs',
         'who_link': True,
-        'what': 'Clone form, fixed!',
-        'link': 'https://gitcoin.co/issue/gitcoinco/code_fund_ads/572/3247',
+        'what': 'Pay Practitioner Implemented!',
+        'link': 'https://gitcoin.co/issue/RibbonBlockchain/IncentivesMVP/16/3282',
         'link_copy': 'View more',
     }, {
-        'who': 'enieber',
+        'who': 'samsparsky',
         'who_link': True,
-        'what': 'README: Updated!',
-        'link': 'https://gitcoin.co/issue/sigillabs/mobidex/236/3166',
+        'what': 'Token Buyback for Gnosis.',
+        'link': 'https://gitcoin.co/issue/GnosisEcosystemFund/Gnosis-Bounties-/3/1626',
         'link_copy': 'View more',
     }, {
-        'who': 'gutsal-arsen',
+        'who': 'PierrickGT',
         'who_link': True,
-        'what': 'Safe Keyboard, Implemented',
-        'link': 'https://gitcoin.co/issue/polkawallet-io/polkawallet-RN/104/2944',
+        'what': 'The edit button now works.',
+        'link': 'https://gitcoin.co/issue/gitcoinco/web/4786/3227',
         'link_copy': 'View more',
     }, ]
 
     sponsor = {
-        'name': 'Allinfra',
-        'title': 'The Future of Infrastructure Finance',
+        'name': 'Quantstamp',
+        'title': 'Scan your smart contract with the Quantstamp Security Network V2 today.',
         'image_url': 'https://s3.us-west-2.amazonaws.com/gitcoin-static/jDSk7ZTfpY19PWdwwsk8puNd.png',
-        'link': 'http://bit.ly/Allinfra',
-        'cta': 'Register today',
+        'link': 'http://bit.ly/QStamp',
+        'cta': 'Scan now',
         'body': [
-            'Bringing access, choice, and liquidity to unlisted infrastructure assets using the power of Ethereum'
+            'Reentrancy bu🐜s are easy to miss. Have confidence in your code and integrate security checks as part of your development workflow.🛡'
         ]
     }
 
     bounties_spec = [{
-        'url': 'https://github.com/gitcoinco/web/issues/4744',
-        'primer': 'Allow Ability to Resurface Bounties to the Top When Expiration Date is Extended',
+        'url': 'https://github.com/fluencelabs/Bounties/issues/1',
+        'primer': 'Help Build The Real Decentralized Web',
     }, {
-        'url': 'https://github.com/SpeckleOS/speckle-browser-extension/issues/89',
-        'primer': 'Global Settings (Change Colour Theme and Log Out)',
+        'url': 'https://gitcoin.co/issue/PegaSysEng/BountiedWork/20/3281',
```

Comment:
Remember these have to be github links, not gitcoin @ryan-shea

Reply:
fixed. my bad.